plugins/search/src/components/SearchModal/SearchModal.tsx

```diff
@@ -99,10 +114,11 @@ export const Modal = ({ open = true, toggleModal }: SearchModalProps) => {
       <DialogTitle>
         <Paper className={classes.container}>
           <SearchBarBase
+            {...searchBarProps}
```

Comment:
There is a problem asking whether we should avoid multiple search bar implementations here.
Do you know the context of why we are using `<SearchBarBase />` instead of `<SearchBar />` @emmaindal ?
The `SearchBar` already has a `debounceTime` property ready to use, for example. I would like to know if we can replace here making the `SearchBar` props customizable as the Material UI `TextField` is...